### PR TITLE
docs: add contributing guide link and update clone command

### PR DESCRIPTION
**Pull request by Misaka-L**
Since many tanslator report (to me) that.... they can't find the way to solve the symlinks and git submoduels problem. (because they are unfamiliar with tools like git and didn't find that there was a CONTRIBUTING.md file....)

I made a pull request to add CONTRIBUTING.md link to each README.md. And update the root CONTRIBUTING.md file clone command to make sure that, symlinks is create correctly (instead of a plain text file with file path)

and maybe we need to consider that add a guide for translator to teach them build a develope environment to see translation change real time?

### CODE REVIEW

**Review by anatawa12 — CHANGES_REQUESTED** [review]
Thank you for suggestion!

Adding link to contributing.md is good!

However, I don't like to add core.symlink=true because it's very environment specific.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,5 +31,5 @@ For Windows machines, you need to setup so your current user can create symbolic
 To setup your project, use the following commands.
 
 ```bash
-git clone --recurse-submodules https://github.com/vrc-get/vrc-get
+git clone --recurse-submodules -c core.symlinks=true https://github.com/vrc-get/vrc-get.git
```
[the remark this answers is not in the document]
Comment (Misaka-L) [reply CONTRIBUTING.md]: I removed it. and think maybe we can add a single command to tell user how to enable symlinks support for their git?

**Review by anatawa12 — APPROVED** [review]
LGTM! Thank you!

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (anatawa12) [reply CONTRIBUTING.md]: Sorry I didn't notice the comment  in the resolved thread.

I don't think adding core.symlink=true is the best way to enable symlink and the link above describes correct way to enable symlink and workaround with copying files (per repository core.symlink)

[the remark this answers is not in the document]
Comment (Misaka-L) [reply CONTRIBUTING.md]: > Sorry I didn't notice the comment in the resolved thread.
> 
> I don't think adding core.symlink=true is the best way to enable symlink and the link above describes correct way to enable symlink and workaround with copying files (per repository core.symlink)

After some researches... I think add a link to a wiki article is a better solution due to Windows has a bad symlink support.

**Pull request merged**